source/fundamentals/crud/read-operations/distinct.txt

```diff
@@ -0,0 +1,161 @@
+.. _node-fundamentals-distinct:
+
+========================
+Retrieve Distinct Values
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Use the ``distinct()`` method to retrieve all distinct values for a specified field 
+across a collection.
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+Follow the instructions in the examples below to insert data into a collection and
+return the unique values of a given field. Consider the following collection of 
```

Comment:
issue: I didn't catch this earlier, but looking at the page again I noticed that this sentence doesn't seem to apply to the page since there aren't instructions for inserting data into a collection.